code/game/objects/items/weapons/tools/scalpels.dm

```diff
@@ -28,13 +27,13 @@
 
 /obj/item/weapon/tool/scalpel/laser
 	name = "laser scalpel"
-	desc = "A scalpel augmented with a directed laser, for more precise cutting without blood entering the field."
+	desc = "A scalpel augmented with a directed laser, for more precise cutting without blood entering the field. Can be used as cautery to some extent as well."
```

Comment:
"A scalpel which uses a directed laser to slice instead of a blade, for more precise surgery while also cauterizing as it cuts."